### PR TITLE
Model divya

### PR DESCRIPTION
**Pull request by Divsjains6021**
database connection set up successfully. Tested and working fine

### CODE REVIEW

File: src/main/java/com/propertydekho/infoservice/InfoServiceApplication.java [hunk 1]
```diff
@@ -1,9 +1,12 @@
 package com.propertydekho.infoservice;
 
 import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 
-@SpringBootApplication
+@SpringBootApplication(scanBasePackages = "com.propertydekho.infoservice")
+//@EnableAutoConfiguration(exclude = {DataSourceAutoConfiguration.class})
```
Comment (jainsameer1991) [inline src/main/java/com/propertydekho/infoservice/InfoServiceApplication.java]: Please remove the comment @Divsjains6021

Comment (Divsjains6021) [reply src/main/java/com/propertydekho/infoservice/InfoServiceApplication.java]: Resolved conflicts

File: src/main/java/com/propertydekho/infoservice/models/PropertyDetails.java [hunk 2]
```diff
@@ -16,14 +15,29 @@
     private String propName;
     @JsonProperty("prop_price")
     private double propPrice;
+    @JsonProperty("budget_max")
+    private double budgetMax;
```
Comment (jainsameer1991) [inline src/main/java/com/propertydekho/infoservice/models/PropertyDetails.java]: @Divsjains6021 Remove BudgetMax

File: src/main/java/com/propertydekho/infoservice/models/PropertyDetails.java [hunk 3]
```diff
@@ -16,14 +15,29 @@
...
     public PropertyDetails() {
 
     }
 
-    public PropertyDetails(String propID, String propName, double propPrice) {
+    public PropertyDetails(String propID, String propName, double propPrice, double budgetMax, String bedroom, String saleType, String constructionStatus, String area) {
         this.propID = propID;
         this.propName = propName;
         this.propPrice = propPrice;
+        this.budgetMax = budgetMax;
```
Comment (jainsameer1991) [inline src/main/java/com/propertydekho/infoservice/models/PropertyDetails.java]: Remove this budgetMax field

File: src/main/java/com/propertydekho/infoservice/models/PropertyRowMapper.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+import com.propertydekho.infoservice.entity.PropFilterableSortableData;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class PropertyRowMapper implements RowMapper<PropFilterableSortableData> {
+    @Override
+    public PropFilterableSortableData mapRow(ResultSet rs, int arg1) throws SQLException {
```
Comment (jainsameer1991) [inline src/main/java/com/propertydekho/infoservice/models/PropertyRowMapper.java:11]: @Divsjains6021 Where this arg1 variable is used

**Review by jainsameer1991 — APPROVED** [review]
Excellent work!

**Pull request merged**